### PR TITLE
Harmonise release leads table & roster

### PR DESCRIPTION
**Pull request by ReToCode**
# Changes
- Add client & functions column to release leads

I think we should have client & functions also on this table, as you folks are always also a part of the release.

/assign @dsimansk

### CODE REVIEW

**Comment by ReToCode** [comment]
Questions about the roster:
- @evankanderson @lionelvillard do you folks want to stay on the Eventing release roster?
- @rhuss do you want to stay on the client release roster?
- @lkingland could you ask in your WG who would like to be added for functions?

**Comment by dsimansk** [comment]
> I think we should have client & functions also on this table, as you folks are always also a part of the release.

I see the motivation here. However, as far as I remember we (client) were always carried by the appointed release leads from either Serving or Eventing WGs. Hence there are no more special release steps for Client, plugins or Functions repositories. 
 
I wonder if we really need the split to WG groups per historical context. Or we should merge pools of available release leads (I assume usually WG leads apply here) and pick the rotation from a single one.

I would like to discuss it with TOC as well. If there are any opinions.  

/cc @knative/technical-oversight-committee

**Comment by ReToCode** [comment]
> I wonder if we really need the split to WG groups per historical context. Or we should merge pools of available release leads (I assume usually WG leads apply here) and pick the rotation from a single one.

I think that is a very good point and I like your proposal.

**Comment by psschwei** [comment]
If I remember right, part of the rationale for having someone from Serving and Eventing on the release was in case there were Serving- or Eventing-specific issues that popped up, for example, something in the autoscaler that would be difficult for someone not familiar with Serving to fix. 

Not sure how often those kinds of issues are still occurring (if at all), but assuming someone from the WG could be available at some point to assist if/when those types of things pop up, then I could see not needing them to be a formal release lead...

**Comment by dprotaso** [comment]
There's an issue here discussion this point: https://github.com/knative/release/issues/32

tl;dr we just need a roster of folks to help shepherd the release. Any issues with particular repos should then require the release leads to pull in the respective WG leads to help. eg. test is flaking etc.

**Comment by ReToCode** [comment]
So assuming, that in the future we'll only have one or two release leads shepherding the release and folks from the respective WGs joining to take care of their repos.

I updated the PR to reflect:
- The currently still active members of the release rotation
- Updated the table to have only one column for release leads

WDYT?

**Comment by evankanderson** [comment]
I'm willing to stay on a rotation; historically (when this was started) there was a bit of unsteadiness in the releases that I think has been ironed out since.
